package.json

```diff
@@ -24,7 +24,7 @@
   },
   "dependencies": {
     "ember-cli-version-checker": "^1.2.0",
-    "execa": "^0.7.0",
+    "execa": "^0.7.0 || ^0.8.0 || ^0.9.0",
```

Comment:
I think `0.7.0 || < 1.0.0` would be a bit more concise. Also we wouldn't have to update this when e.g. `0.10.0` gets released. This is quite helpful btw if you haven't seen it: https://semver.npmjs.com